flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -37,7 +37,7 @@ activate_cg_limit_based_on_mach = 0 ; Activate cg limitation depending on mach v
 ; MLDW = 67400 kg/148591 lbs
 ; MZFW = 64300 kg/141757 lbs
 ; OEW = 42500 kg/93697 lbs
-; Max Payload = 21800 kg/48060 lbs
```

Comment:
Why was this changed?  You can't have a payload that, when added to the OEW is more than the MZFW. So, max payload == 64300 kg - 42500 kg = 21800 kg. What this means if that you carry 174 passengers with bags at 104 kg/passenger, the cargo weight will be limited to 3704 kg. If you carry fewer passengers (or they weigh less), you can carry more payload.

Is this variable used anywhere?

Reply:
Pushed a fix for this, along with the hard limit for pax, bag and cargo not exceeding max payload